skrf_qtapps/widget_test.py

```diff
@@ -6,7 +6,7 @@
 
 class nwa_dummy():
     def __init__(self, NPORTS=2, NCHANNELS=False):
-        self.NPORTS = NPORTS
```

Comment:
why is nports being changed to lower case but not nchannels?

Reply:
Hmm, I think this was by accident, as something failed. I'll try this again.